modules/apache-airflow/templates/values.yaml

```diff
@@ -2010,9 +2010,9 @@ limits: []
 
 # This runs as a CronJob to cleanup old pods.
 cleanup:
-  enabled: true
+  enabled: false
```

Comment:
This is not needed when we are running celery executor since the worker pods are controlled in a different way than when starting pods for tasks through the kubernetes exector.